Xamarin.Forms.Platform.UAP/MasterDetailControl.cs

```diff
@@ -161,18 +161,12 @@ public Windows.Foundation.Size MasterSize
 		{
 			get
 			{
-				// Use the ActualHeight of the _masterPresenter to automatically adjust for the Master Title
-				double height = _masterPresenter?.ActualHeight ?? 0;
-
-				// If there's no content, use the height of the control to make sure the background color expands.
-				if (height == 0)
-					height = ActualHeight;
-
+				double height = ActualHeight;
```

Comment:
So I kept the original change of always starting with the actual height vs the last master height